source/js/components/petition/petition.jsx

```diff
@@ -406,55 +405,36 @@ export default class Petition extends React.Component {
       return (
         <div>
           <p>{this.props.thankYou}</p>
-          <div className="share-btn-container">
-            <button
-              className="share-progress-btn share-progress-fb"
-              onClick={e => this.fbButtonClicked(e)}
-            >
-              <svg
-                aria-hidden="true"
-                xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 264 512"
+          <div className="share-button-group rectangle stacked">
```

Comment:
Follow our established share button group's DOM structure, classes, and styling.